frontend/src/employee-mobile-frontend/child-attendance/ChildNotesSummary.tsx

```diff
@@ -0,0 +1,170 @@
+// SPDX-FileCopyrightText: 2017-2022 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import React, { useMemo } from 'react'
+
+import { AttendanceChild } from 'lib-common/generated/api-types/attendance'
+import { GroupNote } from 'lib-common/generated/api-types/note'
+import LocalDate from 'lib-common/local-date'
+import RoundIcon from 'lib-components/atoms/RoundIcon'
+import { ContentArea } from 'lib-components/layout/Container'
+import {
+  FixedSpaceColumn,
+  FixedSpaceRow
+} from 'lib-components/layout/flex-helpers'
+import { H2, Label, P } from 'lib-components/typography'
+import colors from 'lib-customizations/common'
+import { fasExclamation, farStickyNote, farUsers } from 'lib-icons'
+
+import { useTranslation } from '../common/i18n'
+
+export default React.memo(function ChildNotesSummary({
+  child,
+  groupNotes
+}: {
+  child: AttendanceChild
+  groupNotes: GroupNote[]
+}) {
+  const { i18n } = useTranslation()
+
+  const hasStickyNotes = child.stickyNotes.length > 0
+  const hasDailyNote = child.dailyNote !== null
+  const activeGroupNotes = useMemo(
+    () =>
+      groupNotes.filter((note) =>
+        note.expires.isAfter(LocalDate.todayInHelsinkiTz())
+      ),
+    [groupNotes]
+  )
+  const hasGroupNotes = activeGroupNotes.length > 0
+
+  if (!hasStickyNotes && !hasDailyNote && !hasGroupNotes) return null
+
+  return (
+    <FixedSpaceColumn spacing="m">
+      {hasStickyNotes && (
+        <ContentArea
```

Comment:
Pitäiskö tässä harrastaa vähän DRYtä (Don't repeat yourself): 
Ensin tämmönen:

```
const NoteSection: React.FC<{
  icon: IconDefinition
  iconColor: string
  title: string
  children: React.ReactNode
}> = ({ icon, iconColor, title, children }) => (
  <ContentArea
    shadow
    opaque={true}
    paddingHorizontal="s"
    paddingVertical="s"
    blue
  >
    <FixedSpaceRow>
      <span>
        <RoundIcon content={icon} color={iconColor} size="m" />
      </span>
      <FixedSpaceColumn spacing="s">
        <H2 noMargin>{title}</H2>
        {children}
      </FixedSpaceColumn>
    </FixedSpaceRow>
  </ContentArea>
)
```

ja sitten käytetään tässä ja parissa kohtaa alempana:
```suggestion
        <NoteSection
          icon={fasExclamation}
          iconColor={colors.accents.a5orangeLight}
          title={i18n.attendances.notes.childStickyNotes}
        >
          {child.stickyNotes.map((gn) => (
            <P key={gn.id} noMargin>
              {gn.note}
            </P>
          ))}
        </NoteSection>
```

Reply:
Yeah true, fixed!